sale_line_returned_qty/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds the returned quantity of sale order lines as a computed field.
```

Comment:
More than a technical explanation, this should be more a functional one (what it does, where it is displayed).

Reply:
Hey @rousseldenis, the description is updated.